<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 10, 10);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(25, 25, 25);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string>Adicionar Cliente</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Nome:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="nome">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(150, 150, 150);</string>
                 </property>
                 <property name="placeholderText">
                  <string>Nome do cliente</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Telefone:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="telefone">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>155</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(150, 150, 150);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                 <property name="dragEnabled">
                  <bool>false</bool>
                 </property>
                 <property name="placeholderText">
                  <string>Somente números</string>
                 </property>
                 <property name="cursorMoveStyle">
                  <enum>Qt::LogicalMoveStyle</enum>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Endereço:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="endereco">
                 <property name="minimumSize">
                  <size>
                   <width>317</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(150, 150, 150);</string>
                 </property>
                 <property name="placeholderText">
                  <string>Endereço do cliente</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>CPF:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="cpf">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(150, 150, 150);</string>
                 </property>
                 <property name="placeholderText">
                  <string>Somente números</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="remove_btn">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 170, 255);</string>
             </property>
             <property name="text">
              <string>Remover</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cancel_edit_btn">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="save_btn">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 85, 255);</string>
             </property>
             <property name="text">
              <string>Salvar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLineEdit" name="pesquisa">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(212, 199, 255);</string>
       </property>
       <property name="placeholderText">
        <string>Digite aqui para buscar um cliente</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lupa">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Pesquisar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 170, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../assets/icons/lupa.png</normaloff>../assets/icons/lupa.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tabela_clientes">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 150);</string>
     </property>
     <column>
      <property name="text">
       <string>Nº</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefone</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Endereço</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CPF</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
